test/Microsoft.ML.Tests/TrainerEstimators/TreeEnsembleFeaturizerTest.cs

```diff
@@ -108,5 +115,672 @@ public void TreeEnsembleFeaturizerOutputSchemaTest()
+                TreesColumnName = null,
+                PathsColumnName = null,
+                LeavesColumnName = "Leaves"
+            };
+
+
+            bool isWrong = false;
+            try
+            {
+                var wrongPipeline = ML.Transforms.FeaturizeByFastTreeBinary(options).
+                    Append(ML.Transforms.Concatenate("CombinedFeatures", "Features", "Trees", "Leaves", "Paths")).
+                    Append(ML.BinaryClassification.Trainers.SdcaLogisticRegression("Label", "CombinedFeatures"));
+                var wrongModel = wrongPipeline.Fit(dataView);
+            }
+            catch
+            {
+                isWrong = true; // Only "Leaves" is produced by tree featurizer, so accessing "Trees" and "Paths" may lead to an error.
+            }
+            Assert.True(isWrong);
+
+            var pipeline = ML.Transforms.FeaturizeByFastTreeBinary(options).
+                Append(ML.Transforms.Concatenate("CombinedFeatures", "Features", "Leaves")).
+                Append(ML.BinaryClassification.Trainers.SdcaLogisticRegression("Label", "CombinedFeatures"));
+            var model = pipeline.Fit(dataView);
+            var prediction = model.Transform(dataView);
+            var metrics = ML.BinaryClassification.Evaluate(prediction);
+
+            Assert.True(metrics.Accuracy > 0.98);
+            Assert.True(metrics.LogLoss < 0.05);
+            Assert.True(metrics.AreaUnderPrecisionRecallCurve > 0.98);
+        }
+
+        [Fact]
+        public void TreeEnsembleFeaturizingPipelineMulticlass()
+        {
+            int dataPointCount = 200;
+            var data = SamplesUtils.DatasetUtils.GenerateFloatLabelFloatFeatureVectorSamples(dataPointCount).ToList();
+            var dataView = ML.Data.LoadFromEnumerable(data);
+
+            var trainerOptions = new FastForestRegressionTrainer.Options
+            {
+                NumberOfThreads = 1,
+                NumberOfTrees = 10,
+                NumberOfLeaves = 4,
+                MinimumExampleCountPerLeaf = 10,
+                FeatureColumnName = "Features",
+                LabelColumnName = "Label",
+                ShuffleLabels = true
+            };
+
+            var options = new FastForestRegressionFeaturizationEstimator.Options()
+            {
+                InputColumnName = "Features",
+                TreesColumnName = "Trees",
+                LeavesColumnName = "Leaves",
+                PathsColumnName = "Paths",
+                TrainerOptions = trainerOptions
+            };
+
+            var pipeline = ML.Transforms.Conversion.Hash("HashedLabel", "Label", numberOfBits: 3).
+                Append(ML.Transforms.FeaturizeByFastForestRegression(options)).
+                Append(ML.Transforms.Concatenate("CombinedFeatures", "Features", "Trees", "Leaves", "Paths")).
+                Append(ML.MulticlassClassification.Trainers.SdcaMaximumEntropy("HashedLabel", "CombinedFeatures"));
```

Comment:
The SDCA will be able to take in the original Labels from the multi-class dataset.
```suggestion
                Append(ML.MulticlassClassification.Trainers.SdcaMaximumEntropy("Label", "CombinedFeatures"));
```